test-network-function/identifiers/identifiers.go

```diff
@@ -269,7 +268,7 @@ func XformToGinkgoItIdentifierExtended(identifier claim.Identifier, extra string
 // it extracts the suite name and test name from a claim.Identifier based
 // on the const url which contains a base domain
 // From a claim.Identifier.url:
-//   http://test-network-function.com/tests-case/SuitName/TestName
+//   http://test-network-function.com/testcases/SuiteName/TestName
 // It extracts SuitNAme and TestName
```

Comment:
If you're fixing the comment in line 272, it would be nice to complete the fix with line 273, too ;)